--- a/wart.py	2020-04-06 17:20:13.000000000 -0700
+++ b/wart.py	2020-04-06 17:18:38.000000000 -0700
@@ -6,10 +6,24 @@
 from plugins.RobotsPlugin import RobotsPlugin
 from plugins.SiteInfoPlugin import SiteInfoPlugin
 import urllib3
+import sys
+import re
 
 urllib3.disable_warnings()
 
-target_urls = ["http://example.com"]
+#target_urls = ["http://reddit.com"]
+if len(sys.argv) < 2:
+    print(f"Usage: {sys.argv[0]} <url-file>")
+    sys.exit(2)
+
+f_in = f"{sys.argv[1]}"
+target_urls = []
+with open(f_in, 'r') as f:
+    target_urls = f.read().splitlines()
+
+# Extract TLD
+p = re.compile(r'\w+.[A-Za-z]{2,3}')
+tld = p.findall(target_urls[0])[0]
 
 results = []
 
@@ -44,8 +58,6 @@
     results.append(target.dict())
 
 
-# print(json.dumps(results, indent=4))
+with open(f"/output/http/{tld}/wart-{tld}.json",'w') as f:
+    f.write(json.dumps(results, indent=4))
 
-f = open("example_output.json", "w")
-f.write(json.dumps(results, indent=4))
-f.close()
